test/schemas/relativeId-showSchemaId.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  $id: 'relativeAddressShowSchemaId',
```

Comment:
could you add some `$ref` to `relativeIdSchemaId` in this schema?

Reply:
Thank you for fast reply and Added and update the PR already.